smac/intensifier/hyperband_budget_calculation.ipynb

```diff
@@ -0,0 +1,73 @@
+{
```

Comment:
Why do we have a jupyter notebook file under the package?

Reply:
Is not needed anymore, will remove